docs/details/commands/pmm-admin.md

```diff
@@ -344,18 +362,43 @@ For example, if you want all collectors except `topmetrics`, specify:
 
 ###### Limit dbStats, collStats and indexStats
 
- All the collectors are enabled by default. However, collStats and indexStats are enabled if there are less than 200 collections across all the databases, excluding the admin, config, and local system databases.
+By default, PMM decides the limit for the number of collections to monitor the collStats and indexStats collectors.
 
-For collStats and indexStats you can:
+If you decide to set another limit you could do that with `--max-collections-limit` parameter.
```

Comment:
```suggestion
You can also set an additional limit for the collStats and indexStats collectors with the --max-collections-limit parameter.
```

Reply:
why are we saying limit only applies to collstats and indexStats, it applies to all four `collstats`, `dbstats`, `indexstats`, and `topmetrics`

Reply:
@JiriCtvrtka what is the right behavior ?

Reply:
@denisok Yes after Puneets correction it is fine.